frontend/plugins/tourism_ui/src/modules/tms/components/ActionMenu.tsx

```
@@ -0,0 +1,72 @@
+        </button>
+      </Popover.Trigger>
+      <Popover.Content
+        className="p-1 w-48 rounded-lg border shadow-lg bg-background"
+        side="bottom"
+        align="end"
+      >
+        {dropdownItems.map((item) => (
+          <div
+            className="flex gap-3 items-center px-4 py-2 w-full text-left rounded-md cursor-pointer hover:bg-muted"
+            onClick={item.onClick}
+          >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)